login.css

```diff
@@ -0,0 +1,198 @@
+@import url('https://vaspacx-components.netlify.app/style.css');
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600&display=swap');
+
+:root{
+ 
+    --mainBrandColor: #3a7ce3;
+    --secondaryDark: #211c1c;
+    --font-primary: #FFFFFF;
+    --lightGrey: rgb(114 110 110);
+    --review-color: #ffb000;
+    --secbg-color: #141212;
+    --logo-color:white;
+ 
+}
+
+.light-theme{
+    --mainBrandColor: #3a7ce3;
+   --secondaryDark: #eef0fd;
+   --font-primary: rgb(0 0 0);
+   --lightGrey: rgb(114 110 110);
+   --review-color: #ffb000;
+   --secbg-color: #e2e0ec;
+   --logo-color:black;
+}
+
+#icon-login{
+    width: 20px;
+    position: absolute;
+    cursor: pointer;
+    top: 4%;
+    right: 2%;
+}
+body{
+    height: 100vh; 
+    margin: 0;
+}
+
+.login-box{
+    box-sizing: border-box;
+    color:var(--font-primary);
+    padding: 0;
+    background-color: var(--secondaryDark);
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
you can use grid-template-columns: repeat(3, 1fr);

Reply:
yes thankyou forgot about that property